cilium-dbg/cmd/encrypt_status.go

```diff
@@ -144,9 +144,44 @@ func getEncryptionMode() {
 	}
 }
 
+func isDecryptionInterface(link netlink.Link) (bool, error) {
+	filters, err := netlink.FilterList(link, tcFilterParentIngress)
+	if err != nil {
+		return false, err
+	}
+	for _, f := range filters {
+		if bpfFilter, ok := f.(*netlink.BpfFilter); ok {
+			if strings.Contains(bpfFilter.Name, "cil_from_network") {
```

Comment:
A comment giving more detail about this filter and why it always indicates a decryption interface could be useful.